imports/ui/components/edit/edit-boolean.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+export default React.createClass({
+  propTypes: {
+    content: React.PropTypes.bool,
+    update: React.PropTypes.func,
+    save: React.PropTypes.func
+  },
+  save () {
+    this.props.save(this.getValue())
+  },
+  update () {
+    this.props.update(this.getValue())
+  },
+  getValue () {
+    return this.refs.radioTrue.checked
+  },
+  render () {
+    return (
+      <form onSubmit={this.save}>
+        <label className='radio-inline'>
+          <input type='radio' name='bool' ref='radioTrue' defaultChecked={this.props.content} onChange={this.update} /> Yes
```

Comment:
Guidance is to use a function to assign the ref here, rather than a string and `this.refs`:

https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute